deploy/images/db/main.sql

```diff
@@ -78,15 +78,15 @@ CREATE TABLE local_ega.main (
        submission_file_size                     BIGINT NULL,
        submission_user                          TEXT NOT NULL, -- Elixir ID, or internal user
  
-       -- Vault information
-       vault_file_reference      TEXT,    -- file path if POSIX, object id if S3
-       vault_file_type           storage, -- S3 or POSIX file system
-       vault_file_size           BIGINT,
-       vault_file_checksum       VARCHAR(128) NULL, -- NOT NULL,
-       vault_file_checksum_type  checksum_algorithm,
+       -- archive information
```

Comment:
probably good to keep the capitalisation for consistency :)
```suggestion
       -- Archive information
```